hazelcast-client/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -23,19 +23,19 @@
 import com.hazelcast.console.LineReader;
 import com.hazelcast.console.SimulateLoadTask;
 import com.hazelcast.core.DistributedObject;
-import com.hazelcast.core.EntryEvent;
-import com.hazelcast.core.EntryListener;
+import com.hazelcast.map.EntryEvent;
```

Comment:
The MultiMap and replicatedMap also make use of EntryEvent. So I'm not sure if this should be moved to com.hz.map.